### PR TITLE
Fix layer choice widget

### PR DESCRIPTION
**Pull request by clementcolin**
An old refacto of Tile.getMesh() to Tile.GetMeshes() was not finished, thus implying some bug. Fixed it in this PR

### CODE REVIEW

**Review by EricBoix — APPROVED** [review]

File: docs/html/Widgets_LayerChoice_views_LayerChoice.js.html [hunk 1]
```diff
@@ -260,7 +260,7 @@ <h1 id='page-title' class="page-title">
         for(let j = 0; j &lt; tiles.length ; j++) {
           if (tiles[j].asAttributeInBatchTable('classe')) {
             let classe = tiles[j].batchTable.content.classe[tiles[j].cityObjects[0].batchId];  
-            htmlTiles += `&lt;p>&lt;input type="checkbox" id="checkbox_${i}_${j}" ${tiles[j].getMesh().visible ? 'checked' : ''}>${classe}&lt;/input>&lt;/p>`;
+            htmlTiles += `&lt;p>&lt;input type="checkbox" id="checkbox_${i}_${j}" ${tiles[j].getMeshes().visible ? 'checked' : ''}>${classe}&lt;/input>&lt;/p>`;
```
Comment (mathieuLivebardon) [inline docs/html/Widgets_LayerChoice_views_LayerChoice.js.html:263]: Why this change?
Also I have the impression that the doc is not up to date in all ways.

**Review by mathieuLivebardon — APPROVED** [review]
For me it's good.
Think to take a look at the focus button in the LayerChoice widget.

File: docs/html/Widgets_LayerChoice_views_LayerChoice.js.html (hunk 1 above)
Comment (clementcolin) [reply docs/html/Widgets_LayerChoice_views_LayerChoice.js.html:263]: It was due to an old refacto, it should not be in this pr, you're right.

Concerning doc, maybe we should reproduce it after a PR ? Or at each UD-Viz published version ?

Comment (mathieuLivebardon) [reply docs/html/Widgets_LayerChoice_views_LayerChoice.js.html:263]: We need to set up a workflow about it. I think updating the doc at each release is a good idea.
